go.mod

```diff
@@ -1,14 +1,14 @@
 module github.com/editorconfig-checker/editorconfig-checker/v3 // x-release-please-major
 
-go 1.23
+go 1.23.0
 
-toolchain go1.23.4
+toolchain go1.24.0
```

Comment:
This happened due to `go mod tidy`
Please make sure this is cool (😎) before merging

Reply:
since we have go1.24 in main now, I would say this is fine. But as soon as I rebased this change vanished anyway.